src/filesystem/impls/filer/FilerFileSystem.js

```diff
@@ -193,11 +193,15 @@ define(function (require, exports, module) {
                     BlobUtils.rename(oldPath, newPath);
                     BrambleEvents.triggerFileRenamed(oldPath, newPath);
                 }
-
-                callback();
+                FileSystemCache.refresh(function(){
+                  if(err){    
+                    return callback(err);
+                  }
```

Comment:
These 3 lines (197-199) need to get indented a further 4-spaces.  Also, line 198 needs to use 4-space indents.  Fix your editor when working on this file to use 4-spaces vs. 2-spaces for your tab indents.